### ISSUE TITLE
Importing @grpc/grpc-js prevents bundling build to produce any output

### ISSUE THREAD

**Issue by kskalski** [rollup/plugins#1059]
- Rollup Plugin Name: @rollup/plugin-commonjs
- Rollup Plugin Version: "@rollup/plugin-commonjs": "22.0.0-1", "@rollup/plugin-node-resolve": "13.0.6",
- Rollup Version: 2.60.1
- Operating System (or Browser): Linux (Ubuntu under WSL)
- Node Version: v12.21.0
- Link to reproduction: https://github.com/kskalski/rollup-protobufjs/tree/grpc

### Expected Behavior

``dist/index.umd.js`` file is produced by build

### Actual Behavior

```
> protobuf-rollup@1.0.0 build
> rollup -c

index.js → dist/index.umd.js...
```

stops at this point without output file being procuced

**Comment by shellscape** [comment]
Thanks for opening an issue. Citing the issue template:

>   🚨 Issues WITHOUT a valid reproduction WILL BE CLOSED!
> 
>   Please provide one by:
>   1. Using the REPL.it plugin reproduction template at https://repl.it/@rollup/rollup-plugin-repro
>   2. Provide a minimal repository link (Read https://git.io/fNzHA for instructions).
>      Please use NPM for installing dependencies!
>      These may take more time to triage than the other options.
>   3. Using the Rollup REPL at https://rollupjs.org/repl/
> 
>   ⚠️ ZIP Files are unsafe and maintainers will NOT download them.

We cannot make this any clearer. Please add a reproduction and we'll be happy to triage further.

**Comment by shellscape** [comment]
Script was a little over-aggressive on that repro link for some reason. Sorry about that.

**Comment by kskalski** [comment]
Fixed in "@rollup/plugin-commonjs": "22.0.0-2"
